task_2/reverse_string.py

```diff
@@ -26,4 +36,7 @@ def reverse_string(string: str) -> str:
             ("", ""),
         ]
     for input_text, expected_result in cases:
-        assert reverse_string(input_text) == expected_result
```

Comment:
whole if __name__ == "__main__": block now can be removed, we have tests

Reply:
It really has sense, thank you!